src/setup/rgw.py

```diff
@@ -251,3 +266,69 @@ def _create_rgw_user(conf):
                 Log.info(f'RGW admin user {user_name} is already created. skipping user creation.')
             else:
                 raise SetupError(rc, f'"{create_usr_cmd}" failed with error {err}.')
+
+    @staticmethod
+    def _create_symbolic_link_fid(conf: MappedConf):
+        """ Create symbolic link of FID sysconfig files."""
+        base_config_path = conf.get(CONFIG_PATH_KEY)
+        sysconfig_file_path = os.path.join(base_config_path, 'rgw', 'sysconfig', Rgw._machine_id)
+        file_name = sysconfig_file_path + '/rgw-0x*'
+        list_matching = []
+        for name in glob.glob(file_name):
+            list_matching.append(name)
+        count = len(list_matching)
+        Log.info(f'rgw FID file count : {count}')
+        if count < 1:
+           raise Exception(f'HARE-sysconfig file is missing at {sysconfig_file_path}')
+
+        # Create symbolic links of rgw-fid files created by hare.
+        # e.g rgw-0x7200000000000001\:0x9c -> rgw-1 , rgw-0x7200000000000001\:0x5b -> rgw-2
+        index = 1
+        for src_path in list_matching:
+            file_name = 'rgw-' + str(index)
+            dst_path = os.path.join(sysconfig_file_path, file_name)
+            Rgw._create_symbolic_link(src_path, dst_path)
+            index += 1
+
+    @staticmethod
+    def _create_symbolic_link(src_path: str, dst_path: str):
+        """create symbolic link."""
+        Log.info(f'symbolic link source path: {src_path}')
+        Log.info(f'symbolic link destination path: {dst_path}')
+        if os.path.exists(dst_path):
+           Log.info('symbolic link is already present')
+           os.unlink(dst_path)
+           Log.info('symbolic link is unlinked')
+        os.symlink(src_path, dst_path)
+        Log.info(f'symbolic link created successfully')
```

Comment:
let this be log.info, but add both soft link & linked filename here

Reply:
updated as above